src/v/datalake/datalake_manager.h

```diff
@@ -47,18 +53,37 @@ class datalake_manager : public ss::peering_sharded_service<datalake_manager> {
     ss::future<> stop();
 
 private:
+    using translator = std::unique_ptr<translation::partition_translator>;
+    using translator_map = absl::btree_map<raft::group_id, translator>;
```

Comment:
does it have to be a `btree` or can we use `chunked_hashmap` ?

Reply:
it doesn't have to be, will switch.